### DISCUSSION TITLE
My Jamulus server only accepts 10 musicians!

### DISCUSSION THREAD

**Discussion by FredNor78** [jamulussoftware/jamulus#1362] [Technical Support & Help] [answered]
Hi,
I created for my choir a local Jamulus server on my PC which runs perfectly except that it only accepts 10 clients connected at the same time; when the 10 first singers are connected, the other ones when looking at the server list see the attached message indicating that my server is full. If one of the connected singers disconnects himself, another one can connect at once, but we never are more than 10
When configuring the server, I never saw any information regarding possible limitation of the number of clients.

Where did I miss something?

![Jamulus Server Full 2021-03-27 095626](https://user-images.githubusercontent.com/81464717/112716164-02c43480-8ee5-11eb-9db9-9ccea80bbac6.png)

**Comment by ann0see** [comment, marked as answer]
You'll need to raise the number of clients:

1. Edit the Jamulus server shortcut (right click; click on properties)
2. Edit the field which contains the path to the executable file and append `-u 20 -T` to enable multi threading and raise the number of supported participants.

See also: https://jamulus.io/wiki/Command-Line-Options

**Reply by FredNor78** [reply]
Many thanks ann0see, I will check this asap

**Reply by FredNor78** [reply]
By the way, could you please indicate how I could suggest modifications in the French version of Jamulus, in other words, could I get in touch with the guys in charge of translating the software from English to French? Thanks!

**Reply by hoffie** [reply]
You can see the list of translators in The Help > About menu. For French, this would be @trebmuh and @jujudusud. Now that we've mentioned them using their Github handles, they should get a notification and will probably read here. So just go ahead and ask (if it's some larger topic, maybe rather in a new discussion as the topic has diverged a bit from your initial enquiry ;)).

**Reply by FredNor78** [reply]
You are right, I would rather open another subject for thism or try to contact them

Thanks again ann0see, and have a nice Easter weekend
